extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -832,45 +832,47 @@ export class AmpStoryPage extends AMP.BaseElement {
    * Plays the given media.
    * @param {!./media-pool.MediaPool} mediaPool
    * @param {!Element} mediaEl
-   * @return {!Promise} Promise that resolves after the media is played.
+   * @return {Promise|void}
```

Comment:
I think we should keep this to always return a promise, handling `Promise|void` is much harder, we don't really need this extra confusion